roles/etcd/tasks/certificates/fetch_server_certificates_from_ca.yml

```diff
@@ -197,8 +197,9 @@
   with_items:
   - "{{ etcd_peer_ca_file }}"
   - "{{ etcd_peer_cert_file }}"
-  - "{{ etcd_peer_key_file }}"
+  - "{{ etcd_peer_cert_file }}"
```

Comment:
Inadvertent change to a duplicate item in `with_items`?

Reply:
Yup, updated.